nautobot/core/settings.py

```diff
@@ -283,40 +284,49 @@
 DEBUG = is_truthy(os.getenv("NAUTOBOT_DEBUG", "False"))
 INTERNAL_IPS = ("127.0.0.1", "::1")
 FORCE_SCRIPT_NAME = None
+
+TESTING = len(sys.argv) > 1 and sys.argv[1] == "test"
```

Comment:
Is the second `sys.argv` always guaranteed be `test`? Can the user chain commands with `nautobot-server...`?
perhaps?
```suggestion
TESTING = len(sys.argv) > 1 and "test" in sys.argv
```

Reply:
It's not guaranteed, for example `nautobot-server --config=my_nautobot_config.py test` would be a valid command that wouldn't trip this check. But this is a pretty standard approach and matches most extant plugins.

I considered the `test in sys.argv` approach but would rather err in the direction of false negatives (not TESTING when we actually are running a test) rather than false positives (setting TESTING when we're actually in a live environment).